advanced/external_references/README.md

```diff
@@ -170,8 +170,8 @@ We are still missing the mesh *RoadPlane*. Copy it from the Project Browser and
 Now we build our scene using the imported resources. Add a Node and a MeshNode as its child.
 Assign the imported material and mesh to the MeshNode. You should see a distorted piece of
 road in the Preview. Adjust the scaling as shown in the picture to match the scale of the
-other objects we are going to import shortly. Finally, rotate the Node -90° on the X-axis,
-this should put the road flat on the ground where it belongs.
+other objects we are going to import shortly. In order to see the street correctly in the 
+Preview you might want to move the camera a bit up and rotate it downwards.
```

Comment:
why make it more vague? "rotate a bit up" is totally ambigous in a 3D scene

Reply:
You are right, we made it more clear and fixed it in the latest commit.